keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -487,6 +487,39 @@ From CPI implementation perspective thet feature can be graduated to beta, as th
 
 Graduating to GA means, that the feature flag checking is removed from the code. It means, that all CPI implementations must be ready to deal with Services with mixed protocol configuration - either rejecting such Services properly or managing the cloud load balancers according to the Service definition.
 
+#### Alpha Graduation
+
+- Feature is implemented and controller with a feature flag. The feature flag is disabled by default.
+
+#### Alpha -> Beta Graduation
+
+- Gather feedback from developers and surveys
+- Complete features A, B, C
+- Tests are in Testgrid and linked in KEP
+
+#### Beta -> GA Graduation
+
+- N examples of real-world usage
+- N installs
+- More rigorous forms of testing—e.g., downgrade tests and scalability tests
+- Allowing time for feedback
+
+**Note:** Generally we also wait at least two releases between beta and
+GA/stable, because there's no opportunity for user feedback, or even bug reports,
+in back-to-back releases.
+
+#### Removing a Deprecated Flag
```

Comment:
Can you remove the boilerplate (like this flag section) and  make sure any text is either clearly "TBD" or has real answers?

Reply:
Yep, added "TBD" everywhere where no content is available yet.

Reply:
Is "Removing a Deprecated Flag" even relevant?